Source/PluginProcessor.h

```diff
@@ -15,6 +15,163 @@
 #include "Tract.hpp"
 #include "WhiteNoise.hpp"
 #include "Biquad.hpp"
+#include <math.h>
+
+
+class PinkTromboneADSR
+{
+public:
+	//==============================================================================
+	PinkTromboneADSR()
+	{
+		recalculateRates();
+	}
+
+	//==============================================================================
+	struct Parameters
+	{
+		Parameters() = default;
+
+		Parameters (float attackTimeSeconds,
+					float decayTimeSeconds,
+					float sustainLevel,
+					float decayExponent)
+			: attack (attackTimeSeconds),
+			  decay (decayTimeSeconds),
+			  sustain (sustainLevel),
+			  decayExp(decayExponent)
+		{
+		}
+
+		float attack = 0.1f, decay = 0.1f, sustain = 1.0f, attackExp=1.0f, decayExp=1.0f;

```

Comment:
I guess the attack exponent is never used (which is fine, but maybe we want to take the variable out in this case.